contracts/deploy/DeployHelper.sol

```diff
@@ -167,42 +160,19 @@ contract DeployHelper {
         stakedTokenProxy.upgradeTo(stakedTokenImplAddress);
         stakedToken = StakedToken(address(stakedTokenProxy));
 
-        stakedToken.configure(
-            StakingAsset(address(trustTokenProxy)),
-            StakingAsset(address(trueUSDProxy)),
-            ProvisionalRegistryImplementation(address(registryProxy)),
-            address(liquidatorProxy)
-        );
+        stakedToken.configure(StakingAsset(address(trustTokenProxy)), StakingAsset(address(trueUSDProxy)), ProvisionalRegistryImplementation(address(registryProxy)), address(liquidatorProxy));
 
         liquidatorProxy.claimProxyOwnership();
         liquidatorProxy.upgradeTo(liquidatorImplAddress);
         liquidator = Liquidator(address(liquidatorProxy));
 
-        liquidator.configure(
-            address(registry),
-            address(trueUSDProxy),
-            address(trustTokenProxy),
-            outputUniswapAddress,
-            stakeUniswapAddress
-        );
+        liquidator.configure(address(registry), address(trueUSDProxy), address(trustTokenProxy), outputUniswapAddress, stakeUniswapAddress);
```

Comment:
Any reason for making these one line?

Reply:
Changed prettify settings a bit, looks good now